index.html

```diff
@@ -12,168 +12,171 @@
 </head>
 
 <body>
-
-
-
-<div class="page_wrapper">
-
-
-  <header class="position_rel">
-    <div class="position_fix green">
-  <div class="section_wrapper section header flex_container">
-    <div class="logo">ReMotely</div>
-    <nav class="nav flex_container">
-      <div class="link_flex_container"><a class="links" href="#">book</a><a class="links" href="#">explore</a>
-        <a class="links" href="#">about</a>
-      </div>
-      <div class="sign_container"><a class="sign" href="#">sign in</a></div>
-    </nav>
-  </div></div>
-</header>
-
-<section class="green_section">
-  <div class=" section_hero">
-    <div class="section_container">
-      <div class="column phrase">
-        <div class="catch_phrase">
-          <h1 class="invite_phrase">Work from <br> a cabin near Helsinki</h1>
-          <div class=button_container>
-            <button class="btn" type="submit">Explore</button>
-          </div>
+  <div class="page_wrapper">
+    <header class="header">
+      <div class="header_wrapper">
+        <div class="section_wrapper section header flex_container">
+          <h2 class="logo">
+            ReMotely
+          </h2>
+          <nav class="nav flex_container">
+            <div class="link_flex_container">
+              <a class="links" href="#">book</a>
+              <a class="links" href="#">explore</a>
+              <a class="links" href="#">about</a>
+            </div>
+            <div class="sign_container">
+              <a class="sign" href="#">sign in</a>
+            </div>
```

Comment:
this should be an unordered list

Reply:
fixed

Reply:
@sinkinshadow you make the same mistake, be careful with it.
✅ 
If I open `<li>`and I put `<a>` inside, then at first I close `</a>`, then `</li>`

❗ 
```html
<li><a>something.      </li></a>
```

Reply:
so fix it ;)

Reply:
fixed what does it change? It is for further coding right? Cause it appears not having any influence for html/css code

Reply:
it has tremendous impact on further code and bugs. It is a bug. You break the box model layout this way and may affect unexpected issues. Closing tags in correct order is a must be 😉

Reply:
got it :) will remember about :)